### PR TITLE
Integrate migration pallet into Development Runtime

### PR DESCRIPTION
**Pull request by mustermeiszer**

### CODE REVIEW

File: runtime/development/src/lib.rs [hunk 1]
```diff
@@ -597,6 +590,22 @@ impl pallet_claims::Config for Runtime {
...
+}
+
+// Implement the migration manager pallet
+// The actual associated type, which executes the migration can be found in the migration folder
+impl pallet_migration_manager::Config for Runtime {
+	type MigrationMaxAccounts = MigrationMaxAccounts;
+	type MigrationMaxVestings = MigrationMaxVestings;
+	type MigrationMaxProxies = MigrationMaxProxies;
+	type Event = Event;
+	type WeightInfo = ();
```
Comment (vedhavyas) [inline runtime/development/src/lib.rs:606]: can you point the standard weight implmentation instead of default ?

**Review by vedhavyas — APPROVED** [review]
LGTM. Use standard weight implelemtation

**Review by mikiquantum — APPROVED** [review]
👍

File: runtime/development/src/lib.rs (hunk 1 above)
Comment (mustermeiszer) [reply runtime/development/src/lib.rs]: Done. Also added this one for altair.

**Pull request merged**